.github/workflows/extra_tests.yml

```diff
@@ -0,0 +1,104 @@
+name: TIM-VX Backend
+
+on:
+  pull_request:
+    branches: [ 4.x ]
+    types: [ labeled, opened, synchronize, reopened ]
+
+
+jobs:
+  x86-simulator-build-test:
+    if: contains(github.event.pull_request.labels.*.name, 'category:dnn_timvx')
+    runs-on: ubuntu-20.04
+    container: yuentau/ocv_ubuntu:20.04
+    env:
+      PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+      SOURCE_BRANCH_NAME: ${{ github.head_ref }}
+      TARGET_BRANCH_NAME: ${{ github.base_ref }}
+    steps:
+      - name: info
+        run: |
+          echo "PR Author: ${{ env.PR_AUTHOR }}""
+          echo "Source branch name: ${{ env.SOURCE_BRANCH_NAME }}"
+          echo "Target branch name: ${{ env.TARGET_BRANCH_NAME }}"
+      - name: clean
+        shell: bash
+        run: find . -mindepth 1 -delete
+      - name: fetch opencv
+        uses: actions/checkout@v3
+        with:
+          repository: opencv/opencv
+          ref: ${{ env.TARGET_BRANCH_NAME }}
+          fetch-depth: 0
+          path: opencv
+      - name: merge opencv with test branch
+        shell: bash
+        run: |
+          cd opencv
+          git config user.email "opencv.ci"
+          git config user.name "opencv.ci"
+          git pull -v "https://github.com/${{ env.PR_AUTHOR }}/opencv" "${{ env.SOURCE_BRANCH_NAME }}" --allow-unrelated-histories
+      - name: configure
+        run: |
+          cmake -B build -DWITH_TIMVX=ON -DCMAKE_INSTALL_PREFIX=./install -DBUILD_SHARED_LIBS=ON -DBUILD_PERF_TESTS=OFF -DBUILD_EXAMPLES=OFF -DBUILD_DOCS=OFF -DWITH_OPENCL=OFF -DBUILD_TESTS=OFF opencv
```

Comment:
I propose to enable tests for CI.

Reply:
Set BUILD_TESTS=ON at the latest commit.